chart/kubeapps/values.yaml

```diff
@@ -292,8 +292,8 @@ authProxy:
   # Image used for the proxy
   image:
     registry: docker.io
-    repository: keycloak/keycloak-gatekeeper
-    tag: 4.6.0.Final
+    repository: bitnami/keycloak-gatekeeper
+    tag: 2.3.0-scratch-r1
```

Comment:
is `scratch-r1` necessary, looks like `2.3.0-r1` should just point to the scratch-based image, right?

Reply:
that's right, that tag is the one that @juan131 pointed me to but it's true that `2.3.0-r1` also uses from scratch.